src/ocm/services/Day2ClusterService.ts

```diff
@@ -142,7 +142,7 @@ const Day2ClusterService = {
       openshiftVersion: ocmCluster.openshift_version,
       // The field "cpu_architecture" is calculated based on the existing hosts' architecture
       // It can be a specific architecture, or "multi" if hosts from several architectures have been discovered
-      cpuArchitecture: ocmCluster.cpu_architecture,
+      cpuArchitecture: transformOcmCpuArchToCpuArch(ocmCluster.cpu_architecture),
```

Comment:
I don't think the transformation should be done here, as here we're just mapping the fields that we want to keep.

We're doing the transformation later, using `mapClusterCpuArchToInfraEnvCpuArch` as mentioned earlier.

Reply:
It we don't do here we have a TSC error